File: backend/src/main/java/com/yuma/app/document/Address.java
package com.yuma.app.document;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import org.springframework.data.mongodb.core.mapping.Document;

@Document
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor

public class Address {

	private int streetAddress;
	private String street;
	private String city;
	private String state;
	private String country;
	private String postalCode;


	@Override
	public String toString() {
		return "Address{" +
			"street address=" + streetAddress +
			", street name=" + street + '\'' +
			", city=" + city + '\'' +
			", state=" + state + '\'' +
			", country=" + country + '\'' +
			", postalCode=" + postalCode +
			'}';

	}

}

File: frontend/src/middleware/objects/Address.js
class Address {

  constructor(streetAddress, street, city, state, country, postalCode) {
    this.streetAddress = streetAddress;
    this.street = street;
    this.city = city;
    this.state = state;
    this.country = country;
    this.postalCode = postalCode;
  }

  toString() {
    return this.streetAddress + this.street + this.city + this.state + this.country + this.postalCode ;
  }
}

export default Address;
